Lib/pdb.py

```diff
@@ -588,6 +588,27 @@ def _hold_exceptions(self, exceptions):
             self._chained_exceptions = tuple()
             self._chained_exception_index = 0
 
+    def process_cmdqueue(self):
+        """Process commands in the command queue.
+
+        The function returns in two cases:
+            1. the command queue is empty
+            2. a command to continue execution is encountered
+
+        The return value is True if a following cmdloop should be executed, and False otherwise.
```

Comment:
I don't think "a follwing cmdloop should be executed" is clear. Can you reword this?

Reply:
I updated both comments and could you see if it makes it clearer?